Signal/src/call/PeerConnectionClient.swift

```diff
@@ -220,6 +224,21 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
         self.videoSender = videoSender
     }
 
+    public func setCameraSource(useBackCamera: Bool) {
+        guard let localVideoSource = self.localVideoSource else {
+            owsFail("\(logTag) in \(#function) localVideoSource was unexpectedly nil")
+            return
+        }
+
+        // certain devices, e.g. 16GB iPod touch don't have a back camera
```

Comment:
Should we add one to our test matrix?

Reply:
Personally, I think this accounts for such a small user base that I'm not inclined to dedicate QA resources to verifying it.